Maps.md

```diff
@@ -0,0 +1,21 @@
+This is a general guide about Maps for all FullScreenShenanigans projects. This guide will explain the concept of Maps, Areas, and Locations within a game and how they're connected.
+
+### Table of Contents
+1. [MapsCreatr](#mapscreatr)
+2. [AreaSpawnr](#areaspawnr)
+
+# MapsCreatr
+
+MapsCreatr is a module that handles the storage of Maps, Areas, and Locations. 
+
+A Map is a collection of Areas. An Area is a 2D space with predefined  `x` and `y` boundaries and Things. Locations are entry points within an Area, where an entry point is where the player ends up after transitioning to a new Area.
```

Comment:
Nit; can you say "player *may* end up"? Otherwise it seems like each area has exactly one entry point.